editor/editor_settings.cpp

```diff
@@ -374,6 +374,11 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	_initial_set("interface/editor/display_scale", 0);
 	// Display what the Auto display scale setting effectively corresponds to.
 	float scale = get_auto_display_scale();
+
+	_initial_set("interface/editor/enable_debugging_pseudolocalization", false);
+	set_restart_if_changed("interface/editor/enable_debugging_pseudolocalization", true);
+	// Use pseudolocalization in editor.
```

Comment:
Remember @akien-mga's comment about using `EDITOR_DEF_RST` in the `TranslationServer` instead of this here.

Reply:
Yeah, using EDITOR_DEF_RST in TranslationServer isn't possible since the setup() function is called before EditorSettings is initialized. So, Godot crashes if I put an EDITOR_DEF_RST in TranslationServer::setup(). Instead, I declared a set_editor_pseudolocalization() function to set the editor_pseudolocalization variable and I am calling that function from EditorSettings::setup_language(). This also solves the issue of caching the editor_pseudolocalization instead of querying it from EditorSettings everytime.